src/websocket/Socket.ts

```diff
@@ -0,0 +1,164 @@
+import { WebSocket } from "ws";
+
+import { Command, Tetris, TetrisStates } from "../core/Tetris";
+
+const DEFAULT_WS_CLOSURE = 1000;
+const DEFAULT_HEARTBEAT_INTERVAL = 30000;
```

Comment:
Not really a huge problem but I suggest we add units after variables the next time for better clarification, i.e. something like `DEFAULT_HEARTBEAT_INTERVAL_MS`